sources/packages/web/Dockerfile.dev

```diff
@@ -22,10 +22,25 @@ RUN npm ci
 COPY . .
 
 # Building app
-RUN npm run build
+RUN npm run build-local
+
+# Set a nginx a user to run the process
+USER nginx
```

Comment:
This line has no impact before line 34 right?  where from nginx is not used? 

should this come before line 46?

Reply:
I removed it as it's not used.